packages/expo-updates/ios/EXUpdates/Update/EXUpdatesBareUpdate.m

```diff
@@ -32,7 +32,7 @@ + (EXUpdatesUpdate *)updateWithBareManifest:(NSDictionary *)manifest
 
   NSMutableArray<EXUpdatesAsset *> *processedAssets = [NSMutableArray new];
 
-  NSString *bundleKey = [NSString stringWithFormat:@"bundle-%@", commitTime];
+  NSString *bundleKey = [NSString stringWithFormat:@"bundle-%@", uuid.UUIDString];
```

Comment:
UUIDString uses all caps I think. Is there a way to just use the opaque value from the manifest without any client side transformations?